### PR TITLE
Use TestCase from superdesk.tests

### PR DESCRIPTION
**Pull request by naspeh**
Regarding https://github.com/superdesk/superdesk-core/pull/566

### CODE REVIEW

**Comment by petrjasek** [comment]
:+1:

File: server/test_factory.py [hunk 1]
```diff
@@ -1,82 +0,0 @@
...
-    :param context: test context
-    :param dict user: user
-    """
-    ad_user = user or test_user
-
-    '''
-    This is necessary as test_user is in Global scope and del doc['password'] removes the key from test_user and
-    for the next scenario, auth_data = json.dumps({'username': ad_user['username'], 'password': ad_user['password']})
-    will fail as password key is removed by del doc['password']
-    '''
```
Comment (jerome-poisson) [inline server/test_factory.py]: should be a comment not a docstring here

Comment (naspeh) [inline server/test_factory.py]: this file is deleted :)

Comment (jerome-poisson) [inline server/test_factory.py]: oh, I should probably consult for color blindness :D

Comment (naspeh) [inline server/test_factory.py]: but yep, should change in superdesk-core [superdesk/tests/__init__.py#L204](https://github.com/superdesk/superdesk-core/blob/master/superdesk/tests/__init__.py#L204)

**Comment by jerome-poisson** [comment]
:+1:

**Pull request merged**